app/code/Magento/Backend/Controller/Adminhtml/Index/GlobalSearch.php

```diff
@@ -21,16 +26,33 @@ class GlobalSearch extends \Magento\Backend\Controller\Adminhtml\Index
     protected $_searchModules;
 
     /**
+     * modules that support preview
+     *
+     * @var array
+     */
+    private $previewModules;
+
+    /**
+     * @var ItemsFactory
+     */
+    private $itemsFactory;
+
+    /**
      * @param \Magento\Backend\App\Action\Context $context
      * @param \Magento\Framework\Controller\Result\JsonFactory $resultJsonFactory
      * @param array $searchModules
+     * @param array $previewModules
      */
     public function __construct(
         \Magento\Backend\App\Action\Context $context,
+        ItemsFactory $itemsFactory,
```

Comment:
No, @ishakhsuvarov means last in a list of all existing arguments (after all existing arguments).
And each new argument added should be optional (nullable)